### PR TITLE
[FUSEQE-2178] create import/export tests with OpenAPI re…

### PR DESCRIPTION
**Pull request by AnetaCadova**
//test: `@import-export-open-api`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mmuzikar** [comment]
retest this please

File: ui-tests/src/test/resources/features/integrations/integration-import-export-open-api.feature [hunk 1]
```diff
@@ -0,0 +1,85 @@
+Feature: API Provider Integration - Import Export
```
Comment (mmuzikar) [inline ui-tests/src/test/resources/features/integrations/integration-import-export-open-api.feature]: Please add proper tags such as `@ui,@api-provider` and the maintainer comment (see in other feature files)

File: ui-tests/src/test/resources/features/integrations/integration-import-export-open-api.feature [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    # now we have exported integration, we can clean state and try to import
+    And clean application state
+    And log into the Syndesis
+    And navigate to the "Integrations" page
+    And click on the "Import" link
+    Then import integration "TODO Integration get single"
+
+    When navigate to the "Integrations" page
+    Then Integration "TODO Integration get single" is present in integrations list
+    Then wait until integration "TODO Integration get single" gets into "Stopped" state
```
Comment (mmuzikar) [inline ui-tests/src/test/resources/features/integrations/integration-import-export-open-api.feature]: Try to start integration and verify it is working by both checking the response of API Provider and the actions of API connectors are working as expected.

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mmuzikar** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/resources/features/integrations/integration-import-export-open-api.feature [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    And fill in TODO API host URL
+    And click on the "Save" button
+
+    When created connections
+      | Todo connector | todo | Todo connection | no validation |
+    And navigate to the "Home" page
+
+    When create an API Provider integration "TODO Integration get single" from file swagger/connectors/todo.json
+    And select API Provider operation flow Fetch task
+    Then check flow title is "Fetch task"
```
Comment (mmuzikar) [inline ui-tests/src/test/resources/features/integrations/integration-import-export-open-api.feature]: Better for what you are doing would be Create new task, as you are creating new tasks in `Fetch task`

File: ui-tests/src/test/resources/features/integrations/integration-import-export-open-api.feature [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+    And click on the "Edit Integration" link
+    And click on the "Save" link
+    And publish integration
+    And navigate to the "Integrations" page
+    Then Integration "TODO Integration get single" is present in integrations list
+
+    Then wait until integration "TODO Integration get single" gets into "Running" state
+
+    When execute GET on API Provider route i-todo-integration-get-single endpoint "/api/1"
```
Comment (mmuzikar) [inline ui-tests/src/test/resources/features/integrations/integration-import-export-open-api.feature]: The /api/1magically produces task with id 2, this doesn't look like the intended behavior, for sake of future, please rewrite this to make more sense.

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Database | Smoke - Periodic invocation to Insert 
Integration - DB to DB | Read and update operations 
API Provider Integration - Import Export | Create an integration with custom API connector and then export and import the integration 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/resources/features/integrations/integration-import-export-open-api.feature [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+    And publish integration
+    And navigate to the "Integrations" page
+
+    Then Integration "TODO Integration" is present in integrations list
+
+    Then wait until integration "TODO Integration" gets into "Running" state
+    When execute POST on API Provider route i-todo-integration endpoint "/api/" with body '{"id":2,"completed":2,"task":"task2"}'
+    Then verify response has status 201
+    And verify response has body
+
```
Comment (mmuzikar) [inline ui-tests/src/test/resources/features/integrations/integration-import-export-open-api.feature]: Remove empty line before doc string

File: ui-tests/src/test/resources/features/integrations/integration-import-export-open-api.feature [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+       {"id":2,"completed":2,"task":"task2"}
+     """
+
+    And validate that all todos with task "task1" have value completed "1", period in ms: "1000"
+    And validate that all todos with task "task2" have value completed "2", period in ms: "1000"
+
+    And validate that number of all todos with task "task1" is "1"
+    And validate that number of all todos with task "task2" is "1"
+
+
```
Comment (mmuzikar) [inline ui-tests/src/test/resources/features/integrations/integration-import-export-open-api.feature]: Leave just one empty line at the end of the file

File: ui-tests/src/test/resources/features/integrations/integration-import-export-open-api.feature [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+@ui
+@api-provider
+@api-connector
+@import
+@export
+
+Feature: API Provider Integration - Import Export
+
+ # TODO: import/export tests with OpenAPI resource referenced from api-provider-based integration
```
Comment (mmuzikar) [inline ui-tests/src/test/resources/features/integrations/integration-import-export-open-api.feature]: What does this TODO mean? This looks like exactly what you just did

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Connection - CRUD | Create and delete 
Connection - CRUD | Kebab menu 
Connection - CRUD | Kebab menu edit and view 
Integration - DB to DB | Read and update operations 
API Provider Integration - Import Export | Create an integration with custom API connector and then export and import the integration 
Integration info | Check error for integration with the same name 
Integration info | Check error for integration which has same name after update 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**